### PR TITLE
Fixed Issue of Guid.Empty Being Changed

### PR DESCRIPTION
**Pull request by DHemken97**
When Generating a list of GUID the Empty Static member was being changed

### CODE REVIEW

**Comment by PureKrome** [comment]
@DHemken97 heya! I'm just trying to understand what the initial problem is.

Can you show some sample code showing what the current code does _before_ your fix, please?

NOTE: I don't have permission to accept PR's .. but I'm curious to see what's going on/learn ...

**Comment by DHemken97** [comment]
@PureKrome Check The Included Unit Test. When you Created a list Builder Of Type Guid The Static member Guid.Empty Would Be Changed.

File: src/FizzWare.NBuilder/PropertyNaming/PropertyNamer.cs [hunk 1]
```diff
@@ -88,15 +88,18 @@ private static bool IsNullableType(Type type)
 
         protected virtual void SetValue<T>(MemberInfo memberInfo, T obj, object value)
         {
-            if (value != null)
+            if (value == null) return;
+            if (memberInfo.Name == "Empty" && memberInfo.DeclaringType == typeof(Guid)) return;
```
Comment (crmckenzie) [inline src/FizzWare.NBuilder/PropertyNaming/PropertyNamer.cs]: I don't think this is the right approach. The code shouldn't special-case Guid. The root problem is that NBuilder is altering a static readonly member field. The code should be modified to _not_ set members that are readonly and/or static.

Comment (DHemken97) [reply src/FizzWare.NBuilder/PropertyNaming/PropertyNamer.cs]: I started to think this too... This has been changed

**Pull request merged**